litellm/types/integrations/prometheus.py

```diff
@@ -219,23 +219,3 @@ class UserAPIKeyLabelValues(BaseModel):
     exception_class: Optional[str] = None
     status_code: Optional[str] = None
     fallback_model: Optional[str] = None
-
-    class Config:
-        fields = {
```

Comment:
cc @krrishdholakia-  do you know if this is needed ?

Reply:
@krrishdholakia